app/services/search/base_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Search
+  class BaseService
+    def initialize(search_url)
+      @search_url = search_url
+    end
+
+    def call
+      uri = URI.parse(search_url)
+
+      response = Net::HTTP.get_response(uri)
+
+      response.body if response.is_a?(Net::HTTPSuccess)
```

Comment:
There is no error handling in case of error 😬

Reply:
I'll take care of this in #19, as it's required to set the status to `:failed` when there's an error 🙏

Reply:
Let's leave a TODO message, just to be sure I don't forget 🙈